cpp/turbodbc_arrow/Test/tests/arrow_result_set_test.cpp

```diff
@@ -422,21 +430,33 @@ TEST_F(ArrowResultSetTest, MultiBatchConversionTimestamp)
                 }
             } else {
                 ASSERT_OK(builder.Append(i));
-                auto td = boost::posix_time::microseconds(i);
-                auto ts = boost::posix_time::ptime(boost::gregorian::date(1970, 1, 1), td);
+                auto tp = std::chrono::system_clock::time_point{
+                    std::chrono::sys_days{std::chrono::year{1970}/std::chrono::month{1}/std::chrono::day{1}} +
```

Comment:
Default constructing a time point and adding one microsecond prevents manually encoding the epoch.

Reply:
Updated